v_frame/src/plane.rs

```diff
@@ -424,9 +424,12 @@ impl<T: Pixel> Plane<T> {
           for (self_pixel, bytes) in
             self_row.iter_mut().zip(source_row.chunks(2))
```

Comment:
chunks_exact wouldn't help?

Reply:
Ah, I didn't know that method existed. I'll give that a try.